ts/updater/updater.ts

```diff
@@ -40,22 +40,28 @@ export async function start(
   autoUpdater.logger = logger;
   autoUpdater.autoDownload = false;
 
-  interval = global.setInterval(async () => {
-    try {
-      await checkForUpdates(getMainWindow, messages, logger);
-    } catch (error) {
-      logger.error('auto-update: error:', getPrintableError(error));
-    }
-  }, 1000 * 60 * 10); // trigger and try to update every 10 minutes to let the file gets downloaded if we are updating
+  interval = global.setInterval(
+    async () => {
+      try {
+        await checkForUpdates(getMainWindow, messages, logger);
+      } catch (error) {
+        logger.error('auto-update: error:', getPrintableError(error));
+      }
+    },
+    1000 * 60 * 10
```

Comment:
Maybe we should move this into the Constants? and the one below

Reply:
moved tot a const

Reply:
in constants

Reply:
Thanks